postgres-appliance/Dockerfile

```diff
@@ -424,17 +424,9 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         && find /usr/share/python-babel-localedata/locale-data -type f ! -name 'en_US*.dat' -delete \
 \
         && pip3 install filechunkio wal-e[aws,google,swift]==$WALE_VERSION google-crc32c==1.1.2 'protobuf<4.21.0' \
+                'git+https://github.com/zalando-pg/wal-e.git#egg=wal-e' \
```

Comment:
```suggestion
                'git+https://github.com/zalando-pg/wal-e.git#egg=wal-e[aws,google,swift]' \
```

Reply:
And of course, `wal-e[aws,google,swift]==$WALE_VERSION` should be removed

Reply:
it is actually `'wal-e[aws,google,swift] @ git+https://github.com/zalando-pg/wal-e.git'`